controllers/controllers.js

```diff
@@ -0,0 +1,8 @@
+const models=require('../models/models.js')
+
+exports.getTopics =(req,res) => {
+
+    return models.fetchTopics().then((topics)=>{
+       return res.status(200).send(topics);
+    });
```

Comment:
When sending your data you should respond with an object containing that data. In this case on a key of `topics` please. You'll have to update the tests accordingly 👍